test/autorest/generated/headergroup/internal/headergroup/header.go

```diff
@@ -345,7 +358,11 @@ func (HeaderOperations) ResponseDatetimeRFC1123HandleResponse(resp *azcore.Respo
 	if !resp.HasStatusCode(http.StatusOK) {
 		return nil, newError(resp)
 	}
-	return &HeaderResponseDatetimeRFC1123Response{StatusCode: resp.StatusCode}, nil
+	val, err := time.Parse(time.RFC3339, resp.Header.Get("value"))
```

Comment:
Same parse issue.

Reply:
It compiles it just doesnt know how to respond to data returned from the endpoint.

Reply:
Try `time.RFC1123Z`, or possibly the variant without the Z suffix.

Reply:
Yup I just figured out the fix, but I think I'll leave it for the next PR where I fix all the time/date related issues.